package.json

```diff
@@ -59,6 +55,11 @@
     "cross-env": "^7.0.3",
     "datauri": "^4.1.0",
     "dotenv": "^10.0.0",
+    "eslint": "^8.2.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-chai-friendly": "^0.7.2",
```

Comment:
> Do we really need all the plugins to be `dependencies` as well? 🙂

Reply:
Yes

open-terms-archive is meant to be a dev dependency (or dependency) in declarations folders.
In order for eslint to be installed, it has to be a real dependency of the project.

I did not find any other solution. @Ndpnt maybe?

Reply:
See #842, I moved back eslint in `devDependencies`.